### PR TITLE
[Quest] add scripts to 'Call of Water' quest series 

### PR DESCRIPTION
**Pull request by miraco**
Issue: https://github.com/cmangos/issues/issues/2868
Old PR: https://github.com/cmangos/tbc-db/pull/1069/files

Changed the way how 'Corrupt Water Spirit of Water' start their relay script using SCRIPT_COMMAND_SEND_AI_EVENT instead.

Its now working as intended. 
Have to recheck ACID ids.

Their is still one dberror that can happen when 'Corrupt Water Spirit of Water' are dead and you start the "backup" 
quest ID: 1103.

### CODE REVIEW

File: Updates/0361_call_of_water.sql [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+-- Relay Scripts
+DELETE FROM dbscripts_on_relay WHERE id IN (96, 100, 5897);
+INSERT INTO dbscripts_on_relay(id, delay, priority, command, datalong, datalong2, datalong3, buddy_entry, search_radius, data_flags, dataint, dataint2, dataint3, dataint4, datafloat, x, y, z, o, speed, condition_id, comments) VALUES
+-- This relay Script starts when quest 96 is accepted or 5 minutes after the Corrupted Water Spirit got spawned
+(96,0,3,29,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Remove NPC Flags'),
+(96,0,2,20,2,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Start Movement'),
+(96,0,1,35,5,0,0,6748,50,512,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Minor Manifestation of Water - Send AI Event'),
+-- Spawn script
+(100,0,1,35,5,50,0,5897,50,512,0,0,0,0,0,0,0,0,0,0,0,'Call of Water - Corrupt Water Spirit of Water - Send AI Event'),
```
Comment (miraco) [inline Updates/0361_call_of_water.sql:35]: DB Error: 
2022-07-07 17:34:55  DB-SCRIPTS: Process table `dbscripts_on_relay` id 100, command 35 has buddy 5897 not found in range 50 of searcher Creature (Entry: 7007 Guid: 17627) (data-flags 512), skipping.
When they are already dead.

**Comment by miraco** [comment]
@AnonXS i rechecked this quest and imo its ready to merge.

Just need to recheck relay ids.

The one dberror isnt fixable atm.

File: Updates/0361_call_of_water.sql (hunk 1 above)
Comment (AnonXS) [reply Updates/0361_call_of_water.sql:35]: Any reason quests 96 and 100 have to start relay scripts instead of using dbscripts on queststart?

Comment (miraco) [reply Updates/0361_call_of_water.sql:35]: No real reason just less code.
relay 96 gets called on quest end, and from creature_ai_script (5 minutes after spawned)
realy 100 gets called from quest 100 and on creature_movement

**Comment by AnonXS** [comment]
For some reason when accepting 96, they start moving random while playing dead.

grafik

**Pull request closed without merging**